### PR TITLE
Added complete code snippets of building blocks for all SDKs

### PR DESCRIPTION
**Pull request by amulyavarote**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Added full code snippets in documentation page (under building blocks) for dapr.io

## Issue reference

This partially fulfills: #1969

### CODE REVIEW

**Comment by msfussell** [comment]
@amulyavarote 

Here are updates that you need to address
1) Under the .NET Server section you should say "The example below shows an ASP.NET Core controller example. You can also create an ASP.NET Core routing example. See https://github.com/dapr/dotnet-sdk/tree/master/examples/AspNetCore for examples

2) This sentence "To publish a topic you need to run an instance of a Dapr sidecar to use the pubsub Redis component. You can use the default Redis component installed into your local environment." only applies to the Dapr CLI and should be moved to that Tab only. 

Also to clarify why you say use Redis (which is very confusing given this is about RabbitMQ), you should reword this to 

"To publish a topic with the CLI you need to use a pubsub component to send the message. You can do this with the local Redis pubsub component running with an instance of a Dapr sidecar. (Note that you could also use the RabbitMQ pubsub component, but that would then be sending the message to itself)"

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md [hunk 1]
```diff
@@ -585,28 +619,34 @@ dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-g
 
 ```javascript
 //dependencies
-import { DaprServer, DaprClient, CommunicationProtocolEnum } from 'dapr-client'; 
+import { DaprServer, CommunicationProtocolEnum } from 'dapr-client'; 
 
 //code
 const daprHost = "127.0.0.1"; 
+const serverHost = "127.0.0.1";
+const serverPort = "6002";
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md]: Not a fan of these hard coded values 👎 Shouldn't this be 6001 for the message to be sent correctly given this below?

dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 npm start

Comment (msfussell) [reply daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md]: Also out of interest, do you have working code examples for each of these languages?

**Comment by msfussell** [comment]
@amulyavarote @greenie-msft - Can you take a look at this PR? Would be good to get these changes in

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md (hunk 1 above)
Comment (amulyavarote) [reply daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md]: Yes. I have working code examples for each of these languages.

**Comment by greenie-msft** [comment]
@msfussell Yes, for context @amulyavarote is adding back all the dependencies since these examples will run via copy paste.

**Review by greenie-msft — APPROVED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 2]
```diff
@@ -126,54 +132,59 @@ namespace EventService
...
-
 //dependencies
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import io.dapr.client.DaprClient;
 import io.dapr.client.DaprClientBuilder;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.*;
```
Comment (willtsai) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]:
```suggestion
import java.util.Map;
```
